eservice/bin/register-with-ledger.sh

```diff
@@ -71,9 +71,11 @@ function Store {
         --spid ${SPID} \
         --save ${eservice_enclave_info_file} \
         --loglevel warn \
```

Comment:
We should make this dependent on `PDO_DEBUG_LEVEL` to make trouble-shooting easier (the recent addition below of log-file screen certainly also helps in debugging).   That said, i currently have that change also in the (updated) PR #480, so could be done there ...

Reply:
Good point -- ok with your update, I'll leave it as is here.